app/src/utils/data/contributions.ts

```diff
@@ -445,7 +445,7 @@ export function filterContributionGrantData(
     if (grantData && grantMetaData && Object.keys(grantMetaData).length) {
       const metaDataVersionId = metadataId(grantData.metaPtr);
       const { logoPtr, name } = grantMetaData[metaDataVersionId];
-      grantLogo = ptrToURI(logoPtr, grantLogo);
+      grantLogo = (logoPtr && logoPtr.protocol === 1 ? ptrToURI(logoPtr) : false) || grantLogo;
```

Comment:
This should also work right? `ptrToUri` will return null if the protocol is not 1
```suggestion
      grantLogo = ptrToURI(logoPtr) || grantLogo;
```

Reply:
I think you're right! Before I had this setup as `IpfsImg` and figured the component didn't need to be so specific, this is a remnant! I'll try it out and get it committed, thanks!